### PR TITLE
Remove Numeric output from Sentiment endpoint

### PR DESCRIPTION
**Pull request by GermanParra**
## Description
https://www.loom.com/share/243829d1f850495080bf0cb146562e0d

Eliminate the posibility for numerical output from the /sentiment end point in the API, making modifications on vader_sentiment and api scripts.

## Type of change

- [x] Bug fix

## Checklist:

- [x] My code follows PEP8 style guide
- [x] I have made corresponding changes to the documentation if necessary
- [x] My changes generate no errors
- [x] No commented-out code
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Comment by ad-victoriam-101** [comment]
This is not a bug this is a requested feature from [This ticket](https://trello.com/c/h5AS5v84/285-as-a-data-science-developer-i-want-the-option-to-classify-texts-sentiment-in-either-human-terms-ie-positive-or-negative-or-with)

This was done to allow the API to get the numeric for future NLTK processes.

File: app/api.py [hunk 1]
```diff
@@ -245,22 +245,12 @@ async def financial_aid(profile_id: str):
```
Comment (BrokenShell) [inline app/api.py:245]: This looks fantastic! You made quick work of this task, way to go! If we need numeric output we can always create a second function for that.

**Review by BrokenShell — APPROVED** [review]
The change to the endpoint looks good too.

**Review by jinjahninjah — CHANGES_REQUESTED** [review]
Looks great German! Thanks for cleaning up some old code, this is a very handy skill and practice makes perfect. 

The only changes to make are in the description of the PR. 
- Remove the boilerplate code: "Fixes # (issue)", "Delete options that are not relevant", and the like. 
- Since we are changing the API, go ahead and change the version number to 0.45.1 and we should be good to go. 

Other than the minor changes, great job on your first PR 👍

**Comment by mikayla-kosmala** [comment]
Nice job on getting the first pull request done for the team!

**Review by mikayla-kosmala — APPROVED** [review]
Nice Job!

**Review by merinakarki — APPROVED** [review]

**Review by jinjahninjah — APPROVED** [review]

File: app/api.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 API = FastAPI(
     title='Underdog Devs DS API',
-    version="0.44.4",
+    version="0.45.1",
```
Comment (merinakarki) [inline app/api.py:14]: Awesome work. You catched this and updated the version.

**Review by merinakarki — APPROVED** [review]

**Review by kevinbui100 — COMMENTED** [review]
The code looks fantastic!

**Pull request merged**